e2e/cypress/integration/messaging/message_edit_post_with_attachment_spec.js

```diff
@@ -13,19 +13,22 @@
 import * as TIMEOUTS from '../../fixtures/timeouts';
 
 describe('MM-13697 Edit Post with attachment', () => {
+    let townsquareLink;
+
     before(() => {
-        // # Login, set display preference and go to /
-        cy.apiLogin('user-1');
-        cy.apiSaveMessageDisplayPreference();
```

Comment:
We used to do that before since other tests are making change to such user-1's preference. But now we're testing in isolation by creating new user every time, there's no need to update it.